src/util/signature_utils.js

```diff
@@ -362,13 +362,47 @@ function authenticate_request_by_service(req, sdk) {
 }
 
 // mutual code of S3, STS and IAM services
-function authorize_request_account_by_token(token, requesting_account, is_secret_optional) {
-    const signature_secret = token.temp_secret_key || requesting_account?.access_keys?.[0]?.secret_key?.unwrap();
-    const should_check_signature = !is_secret_optional || signature_secret;
-    if (should_check_signature) {
-        const signature = get_signature_from_auth_token(token, signature_secret);
-        if (token.signature !== signature) throw new RpcError('SIGNATURE_DOES_NOT_MATCH', `Signature that was calculated did not match`);
+// 1 - check access key status (in case deactivated property true)
+// 2 - check secret_key signature (in case we should check the signature)
+function authorize_request_account_by_token(token, requesting_account) {
```

Comment:
@nimrod-becker This is the function I'm changing that is mutual to all services (S3, STS, and IAM).
It doesn't add new functionality in the containerized environment.